cli/src/main/kotlin/commands/EvaluatorCommand.kt

```diff
@@ -96,6 +97,14 @@ object EvaluatorCommand : CommandWithHelp() {
     )
     private var repositoryConfigurationFile: File? = null
 
+    @Parameter(
+        description = "A file containing the curations. If set the given curations override the curations of the " +
+                "input ort file.",
```

Comment:
This is unclear about whether *all* curations from the input file are overridden, or only those that are *also* present in the file specified by this parameter. Could you please elaborate?

Reply:
I've adjusted the description...